en/micro-integrator/docs/setup/deployment/deploying_wso2_ei.md

```diff
@@ -24,6 +24,18 @@ hostname = "localhost"
 
 Find more [parameters](../../../references/config-catalog/#deployment) for deployment settings.
 
+!!! info "Enable heartbeat messaging"
+    When a client is connected to the broker, both the broker and the client should be able to detect problem situations where the TCP connection is half open or where the connecting client/broker is unresponsive. This can be achieved by enabling heartbeat messaging between the broker and the client.
```

Comment:
we need to reword this.. this is between micro integrator nodes. Broker doesn't comes into play here.